src/components/Button/Button.styles.ts

```diff
@@ -2,13 +2,114 @@ import styled from 'styled-components';
+            fontSize: '10px',
+            height: '24px',
+            lineHeight: '9px',
+        },
+        sm: {
+            padding: '10px 16px',
+            focusPadding: variant === 'secondary' ? '9px 15px' : '8px 14px',
+            fontSize: '12px',
+            height: '32px',
+            lineHeight: '10px',
+        },
+        md: {
+            padding: '12px 20px',
+            focusPadding: variant === 'secondary' ? '11px 19px' : '10px 18px',
+            fontSize: '14px',
+            height: '40px',
+            lineHeight: '14px',
+        },
+        lg: {
+            padding: '14px 24px',
+            focusPadding: variant === 'secondary' ? '13px 23px' : '12px 22px',
+            fontSize: '16px',
+            height: '48px',
+            lineHeight: '17px',
+        },
+    };
+
+    // inverted style color map
+    const invertedColorMap: ButtonColorMap = {
+        primary: {
+            default: colors.primaryText,
+            hover: colors.primaryText,
+            active: colors.primaryText,
+            disabled: colors.centerChannel_8,
+        },
+        secondary: {
+            default: 'transparent',
+            hover: colors.primaryText_8,
+            active: colors.primaryText_16,
+            disabled: 'transparent',
+        },
+        tertiary: {
+            default: colors.primaryText_8,
+            hover: colors.primaryText_12,
+            active: colors.primaryText_16,
+            disabled: colors.centerChannel_8,
+        },
+        quaternary: {
+            default: 'transparent',
+            hover: colors.primaryText_8,
+            active: colors.primaryText_12,
+            disabled: 'transparent',
+        },
+        text: {
+            default: 'transparent',
+            hover: 'transparent',
+            active: colors.primaryText_16,
+            disabled: 'transparent',
+        },
+    };
+
+    // destructive style color map
+    const destructiveColorMap: ButtonColorMap = {
+        primary: {
+            default: colors.error,
+            hover: colors.error,
+            active: colors.error,
+            disabled: colors.centerChannel_8,
+        },
+        secondary: {
+            default: 'transparent',
+            hover: colors.error_8,
+            active: colors.error_8,
+            disabled: 'transparent',
+        },
+        tertiary: {
+            default: colors.error_8,
+            hover: colors.error_12,
+            active: colors.error_16,
+            disabled: colors.centerChannel_8,
+        },
+        quaternary: {
+            default: 'transparent',
+            hover: colors.error_8,
+            active: colors.error_12,
+            disabled: 'transparent',
+        },
+        text: {
+            default: 'transparent',
+            hover: 'transparent',
+            active: colors.error_16,
+            disabled: 'transparent',
+        },
+    };
+
+    // default color map
```

Comment:
What's the need for adding a comment above every variable, especially when the comment and the variable name is same? I think we should remove all such comments.

Reply:
Removed obvious ones